### PR TITLE
Geojson prep

### PR DESCRIPTION
**Pull request by hannahker**
Addressing #8 and #10. Eventually I'd like to move the geojson data to some external Azure store ([for example](https://learn.microsoft.com/en-ca/azure/app-service/configure-connect-to-azure-storage?tabs=basic%2Cportal&pivots=container-linux)), but for now we can update against what is dumped in Azure [here](https://portal.azure.com/#view/Microsoft_Azure_Storage/ContainerMenuBlade/~/overview/storageAccountId/%2Fsubscriptions%2F3767353a-61dc-440a-b098-a362318edbe0%2FresourceGroups%2FIMB-CHD-DataScience-EastUS2%2Fproviders%2FMicrosoft.Storage%2FstorageAccounts%2Fimb0chd0dev/path/projects/etag/%220x8DC6DE2C05EF5EB%22/defaultEncryptionScope/%24account-encryption-key/denyEncryptionScopeOverride~/false/defaultId//publicAccessVal/None). 

When new countries are added, we'll just need to run this script locally and push the changes to the geojson outputs to GitHub.

### CODE REVIEW

File: download_geodata.py
Comment (t-downing) [inline download_geodata.py:1]: What would you think about actually keeping the rest of the CODAB processing (i.e., the stuff that I mentioned [here](https://github.com/OCHA-DAP/ds-floodexposure-monitoring/pull/5#discussion_r1852635431)) here instead?

I was just thinking that this type of CODAB processing (grabbing the EN/FR name, and making a GeoJSON just for the outline) is only really relevant for the app, whereas the outputs of the pipeline repo would be used for other analyses too. So would just keep the app-relevant stuff in this repo, and keep the pipeline one limited to Floodscan processing

Comment (hannahker) [reply download_geodata.py:1]: Ok fair point! I was initially thinking to move all pipelines to the other repo but agree that those are really just app-specific and so probably would be more fitting here.

Comment (t-downing) [reply download_geodata.py:1]: Hmm yeah good point. As discussed in Slack ultimately (I hope) all this `codab` functionality will ultimately be housed elsewhere, so for now some copy+pasting is ok. Then at least each repo can do what it needs to do standalone

Comment (hannahker) [reply download_geodata.py:1]: Ok added in the latest commit!

**Review by t-downing — APPROVED** [review]
Looks good, thanks!

**Pull request merged**